GDJS/Runtime/gd.js

```diff
@@ -85,18 +107,30 @@ gdjs.randomWithStep = function(min, max, step) {
 
 /**
  * Convert an angle in degrees to radians.
+ * @param {number} angleInDegrees The angle in degrees.
+ * @returns {number}
  */
 gdjs.toRad = function(angleInDegrees) {
   return (angleInDegrees / 180) * 3.14159;
 };
 
 /**
  * Convert an angle in radians to degrees.
+ * @param {number} angleInRadians The angle in radians.
+ * @returns {number}
  */
 gdjs.toDegrees = function(angleInRadians) {
   return (angleInRadians * 180) / 3.14159;
 };
 
+/**
+ * A Constructor for a {@link gdjs.RuntimeObject}.
+ * @name ObjectCtor
```

Comment:
Let's maybe call this RuntimeObjectCtor - or even RuntimeObjectConstructor.

Reasons:
* "Runtime" for consistency with RuntimeObject.
* Object alone is too close to a JS "Object" and could creator confusion.
* "Constructor" => let's use full terms.